README.md

```diff
@@ -1,3 +1,74 @@
-### dependency-metrics ###
+## Welcome to dependency-metrics ##
 
-This package is used to monitor how up to date dependencies of a project are.
+### Summary
+A command line tool to analyze and monitor how up-to-date dependencies of a project are. For more detail, see the [Usage](#usage) section below.
+
+#### Supported Package Managers
+- pip
+- yarn
+
+#### Supported Application Monitoring Platforms
+- DataDog
+
+
+### Usage
+Run the `dependency-metrics` tool from within the repository you wish to generate metrics for.
+
+#### Default behavior
+
+All that is needed is the package manager you want to generate metrics for, and `dependency-metrics` will output a table detailing
+how out-of-date each dependency is.
+
+```commandline
+$ dependency-metrics pip
+Behind   Package                      Latest       Version
+0.1.0    my-depedency                 1.1.0        1.0.0
+2.0.0    my-other-dependency          3.2.1        1.2.3
+```
+
+#### --stats option
+
+Use the `--stats` option to generate a simple dictionary that displays the total number of outdated dependencies, as well as a breakdown detailing the number of outdated dependencies for each version type.
+
+```commandline
+$ dependency-metrics pip --stats
+Outdated: 1

```

Comment:
Should this sat `Outdated: 2`, given the lines below?

Reply:
Yep! 02889d21cf3c07e8ca677b1267aa2f9045a3adb7